README.md

```diff
@@ -1,89 +1,87 @@
-# IPFS Specs
+Specifications
+==============
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
 [![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 
-This repository contains the specs for the IPFS Protocol and associated
-subsystems. Some day we will hopefully transform these specs into RFCs.
-For now, they assume a high level of familiarity with the concepts.
+![](media-artifacts/ipfs-splash.png)
 
-
-![](ipfs-splash.png)
+> This repository contains the specs for the IPFS Protocol and associated subsystems. Some day we will hopefully transform these specs into RFCs. For now, they assume a high level of familiarity with the concepts.
```

Comment:
Better to have this paragraph beneath, just not with a >. Put the > above it, the one you corrected it to.